### PR TITLE
47574_course_delete_close

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

File: app/controllers/supervisor/courses_controller.rb [hunk 1]
```diff
@@ -36,6 +45,12 @@ def show
     @course = Course.find(params[:id])
   end
 
+  def destroy
+    @course.destroy
+    flash[:success] = t :course_delete
+    redirect_to supervisor_courses_path
+  end
+
   private
```
Comment (shakil-fg) [inline app/controllers/supervisor/courses_controller.rb]: remove empty line

**Comment by shakil-fg** [comment]
OK

File: app/controllers/supervisor/courses_controller.rb [hunk 2]
```diff
@@ -9,11 +9,20 @@ def edit
   end
 
   def update
-    if @course.update_attributes course_params
-      flash[:success] = t :course_update
-      redirect_to [:supervisor, @course]
+    if params[:type] == "close"
+      if @course.update_attributes status: "close"
```
Comment (manazirahsan) [inline app/controllers/supervisor/courses_controller.rb]: Remove hard quoted string.

**Comment by mdmahmudurrahman** [comment]
OK

**Comment by manazirahsan** [comment]
okey

**Pull request merged**